tests/lib/prepare-project.sh

```diff
@@ -68,17 +68,18 @@ fi
 
 # declare the "quiet" wrapper
 . "$TESTSLIB/quiet.sh"
+. "$TESTSLIB/dirs.sh"
```

Comment:
Could we define that at top-level so that we don't have to source it anywhere? If we place those in `spread.yaml`'s `environment` section of each `system` it will apply everywhere.